content/cucumber/api.md

```diff
@@ -531,9 +531,19 @@ Or changing your JUnit runner class:
 @CucumberOptions(tags = "@smoke and @fast")
 public class RunCucumberTest {}
 ```
-
 {{% /block %}}
 
+## Ignoring a subset of scenarios
```

Comment:
Nice addition. Two things (you might not be aware of):
1) This text block should be below the Javascript and Ruby sections for "Running a subset of scenarios", otherwise it will show the Javascript/Ruby code for that section under this heading.